<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="separator"/>
   <addaction name="actInsert"/>
   <addaction name="actUpdate"/>
   <addaction name="actDelete"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actOpenDB">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/open.bmp</normaloff>:/icons/open.bmp</iconset>
   </property>
   <property name="text">
    <string>打开数据库</string>
   </property>
   <property name="toolTip">
    <string>打开数据库</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/insert.bmp</normaloff>:/icons/insert.bmp</iconset>
   </property>
   <property name="text">
    <string>插入记录</string>
   </property>
   <property name="toolTip">
    <string>插入记录</string>
   </property>
  </action>
  <action name="actUpdate">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/edit.bmp</normaloff>:/icons/edit.bmp</iconset>
   </property>
   <property name="text">
    <string>编辑记录</string>
   </property>
   <property name="toolTip">
    <string>编辑记录</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/delete.bmp</normaloff>:/icons/delete.bmp</iconset>
   </property>
   <property name="text">
    <string>删除记录</string>
   </property>
   <property name="toolTip">
    <string>删除记录</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/exit.bmp</normaloff>:/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
